docs/source/conf.py

```diff
@@ -166,7 +166,18 @@
     ),
 ]
 
-intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+    "distributed": ("https://distributed.dask.org/en/stable", None),
+    "lightgbm": ("https://lightgbm.readthedocs.io/en/latest", None),
+    "matplotlib": ("https://matplotlib.org/stable", None),
+    "numpy": ("https://numpy.org/doc/stable", None),
+    "scipy": ("https://docs.scipy.org/doc/scipy", None),
+    "sklearn": ("https://scikit-learn.org/stable", None),
+    "torch": ("https://pytorch.org/docs/stable", None),
+    "pandas": ("https://pandas.pydata.org/docs", None),
+    "plotly": ("https://plotly.com/python-api-reference", None),
```

Comment:
[Comments to reveiwers]: This might be unnecessary because the links to plotly's API are broken due to https://stackoverflow.com/questions/73528299/link-to-plotly-graph-objects-figure-with-intersphinx. Possibly future `intersphinx` or Plotly fixes this issue, so we can also leave this plotly part as in this PR. Pandas as well.